articles/how-to-install-osquery-and-enroll-macos-devices-into-fleet.md

```diff
@@ -2,11 +2,12 @@
 
 The easiest way to install osquery and enroll macOS devices into your Fleet instance is to use the Fleet osquery installer.
 
-Alternatively, you can run a preview environment of Fleet locally (which automatically adds your device to the locally running Fleet server). Check out the [Getting Started](https://fleetdm.com/get-started) guide for instructions on setting that up.
-
 ## Prerequisites
 
-Before installing osquery on macOS and enrolling that macOS device, you will need to have access to a Fleet server (see [Deploying Fleet on Render](https://fleetdm.com/deploy/deploying-fleet-on-render) for an example.)
+Before installing osquery on macOS and enrolling that macOS device, you will need to have access to
+a Fleet server (see [Deploying Fleet on Render](https://fleetdm.com/deploy/deploying-fleet-on-render) for an example.)
+
+Alternatively, you can try Fleet. Check out the [Get started](https://fleetdm.com/get-started) guide for instructions on setting that up.
```

Comment:
This language is different from the language for linux, but matches Windows. Why is that? I think the linux language is most descriptive.

Reply:
@zhumo no reason that I know of. I agree that Linux is most descriptive. I updated the macOS and Windows language to match Linux.